### ISSUE TITLE
Ugoira -> PNG conversion crashes

### ISSUE THREAD

**Issue by John-Titor** [Nandaka/PixivUtil2#591]
### Prerequisites

* [X] Did you read FAQ section?
* [X] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?

### Description

Ugoira -> PNG conversion is not working with:
```
[Ugoira]
writeugoirainfo = False
createugoira = True
deletezipfile = True
creategif = False
createapng = True
deleteugoira = True
createwebm = False
createwebp = False
```

### Steps to Reproduce

processing ugoira to apng...
Traceback (most recent call last):
  File "/Users/username/work/github/PixivUtil2/PixivUtil2.py", line 186, in download_image
    handle_ugoira(image, filename_save)
  File "/Users/username/work/github/PixivUtil2/PixivUtil2.py", line 914, in handle_ugoira
    PixivHelper.ugoira2apng(ugo_name, gif_filename, __config__.deleteUgoira, image=image)
  File "/Users/username/work/github/PixivUtil2/PixivHelper.py", line 784, in ugoira2apng
    im.append(fImage, delay=delay)
  File "/Users/username/work/github/PixivUtil2/apng.py", line 200, in append
    png = PNG.open(png)
  File "/Users/username/work/github/PixivUtil2/apng.py", line 126, in open
    o.chunks = list(chunks(file))
  File "/Users/username/work/github/PixivUtil2/apng.py", line 66, in chunks
    with io.BytesIO(png) as f:
TypeError: a bytes-like object is required, not 'str'
Error at download_image(): (<class 'TypeError'>, TypeError("a bytes-like object is required, not 'str'"), <traceback object at 0x11c3b1960>) at https://i.pximg.net/img-zip-ugoira/img/2019/12/06/19/06/06/78165891_ugoira1920x1080.zip (9000)

### Versions

d2ad036a96d5b51703920f1a9b607624bcf73f1f (v20191214 plus fix for #588)

**Comment by Nandaka** [comment]
fixed in 2d1a0a3

**Comment by John-Titor** [comment]
Confirmed fixed with 88a3db7 and apng 0.3.3
